src/workspace/shared/StatusLayout.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```

Comment:
This is also present in the /src/shared directory

Reply:
Updated.